### DISCUSSION TITLE
Momentary function impossible - aubess mini smart switch

### DISCUSSION THREAD

**Discussion by kalagan55** [Koenkk/zigbee2mqtt#17218] [Devices]
Hello,
I cannot configure my aubess mini smart switch with the momentary function (one-second pulse like a push button).
I activate the option but it works like troggle. I use home assistant with zigbee2mqtt.
![image](https://user-images.githubusercontent.com/100440818/229345889-be26b95e-2a67-4bd6-8795-2d715267632f.png)
![image](https://user-images.githubusercontent.com/100440818/229345912-d288c12c-84f3-4fb2-8510-8750dcc6fa6b.png)

**Comment by kalagan55** [comment]
no response ?

**Comment by nysmag** [comment]
I have the same issue.
Pressing two times within ~4sec changes the state.
Can't see the device sending anything on single push, so maybe not supported?
